gen/defenses/armor_group_generator.py

```diff
@@ -12,36 +12,81 @@
 )
 
 
-def generate_armor_group(faction: str, game, ground_object):
+def generate_armor_group(
+    faction: str, game, ground_object, shorad: bool, player_wants_shorad: bool
+):
     """
     This generate a group of ground units
     :return: Generated group
     """
+    include_shorad: bool = False
+    if shorad == True and player_wants_shorad == True:
+        include_shorad = True
+
     armor_types = (
-        GroundUnitClass.Apc,
         GroundUnitClass.Atgm,
-        GroundUnitClass.Ifv,
         GroundUnitClass.Tank,
+        GroundUnitClass.Apc,
+        GroundUnitClass.Ifv,
```

Comment:
I can't tell what this changes. Add a comment?

Reply:
Ping.

Reply:
Was splitted for the light armor groups, then re-added so that the behavior is the same as before.